src/pages/index.js

```diff
@@ -24,24 +24,191 @@ const profileDescriptionInput = document.querySelector(
   "#profile-description-input"
 );
 
-/* Funtions */
+/* Api Endpoints */
+const userProfile = new UserInfo(
+  ".profile__title",
+  ".profile__description",
+  ".profile__image"
+);
+
+/* Form Validators */
+const profileEditForm = profileEditPopup.querySelector(".popup__form");
+const addCardForm = addCardPopup.querySelector(".popup__form");
+const avatarForm = document.querySelector("#avatar-edit-popup .popup__form");
+
+const profileEditFormValidator = new FormValidator(settings, profileEditForm);
+const addCardFormValidator = new FormValidator(settings, addCardForm);
+const avatarFormValidator = new FormValidator(settings, avatarForm);
+
+profileEditFormValidator.enableValidation();
+addCardFormValidator.enableValidation();
+avatarFormValidator.enableValidation();
+
+/* Popup Instances */
+const deleteCard = new PopupWithFormSubmit("#trash-popup");
+const imagePopup = new PopupWithImage("#image-popup");
+const avatarPopup = new PopupWithForm("#avatar-edit-popup", handleAvatarSubmit);
+const profilePopup = new PopupWithForm(
+  "#profile-edit-popup",
+  handleProfileEditSubmit
+);
+const cardPopup = new PopupWithForm("#add-card-popup", handleAddCardFormSubmit);
+
+deleteCard.setEventListeners();
+imagePopup.setEventListeners();
+avatarPopup.setEventListeners();
+profilePopup.setEventListeners();
+cardPopup.setEventListeners();
+
+/* Get Initial Cards */
+let cardList;
+api
+  .getInitialCards()
+  .then((initialCards) => {
+    console.log("intialCards", initialCards);
+    cardList = new Section(
+      {
+        items: initialCards,
+        renderer(item) {
+          const cardElement = renderCard(item);
+          this._container.prepend(cardElement);
```

Comment:
Only class `Section` should add any card to the DOM.

You need to create `prependItem` inside `Section` to add the card with `prepend`